spec/models/mixins/tenant_quotas_mixin_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe TenantQuotasMixin do
+  before do
+    Tenant.seed
+  end
+
+  let(:root_tenant) do
+    Tenant.root_tenant
+  end
+
+  let(:super_admin_role)  { FactoryGirl.create(:miq_user_role, :features => MiqProductFeature::SUPER_ADMIN_FEATURE) }
+  let(:tenant_admin_role) { FactoryGirl.create(:miq_user_role, :features => MiqProductFeature::TENANT_ADMIN_FEATURE) }
+
+  let(:tenant_1)   { FactoryGirl.create(:tenant, :parent => root_tenant) }
+  let(:tenant_1_1) { FactoryGirl.create(:tenant, :name => 'Tenant 2', :parent => tenant_1) }
+  let(:tenant_1_2) { FactoryGirl.create(:tenant, :name => 'Tenant 3', :parent => tenant_1, :divisible => false) }
+
+  let(:group_tenant_1_tenant_admin) { FactoryGirl.create(:miq_group, :miq_user_role => tenant_admin_role, :tenant => tenant_1) }
+  let(:user_tenant_1_tenant_admin)  { FactoryGirl.create(:user, :userid => 'user', :miq_groups => [group_tenant_1_tenant_admin]) }
```

Comment:
Should `:userid` be moved to a sequence in the factory?

Reply:
there is sequence for, so removing it from here